go.py

```diff
@@ -236,31 +234,31 @@ def add_stone(self, color, c):
 
         return captured_stones
 
-    def _create_group(self, color, c, liberties):
+    def _merge_from_played(self, color, played, libs, other_group_ids):
+        stones = set()
```

Comment:
initialize this as `stones = set([played])`?

Reply:
If you do this you can't include the assert .isdisjoint() below. but I'm taking that out so easy change.